accounts/account-router.js

```diff
@@ -0,0 +1,55 @@
+const express = require('express');
+const {getAllAccounts, getAccountById, addNewAccount, changeAccountById, removeAccount} = require('./account-model');
+const router = express.Router();
+
+router.get('/', async(req,res)=>{
+    try{
+        const allAccounts = await getAllAccounts();
+        res.status(200).json(`${allAccounts} account was deleted` );
+    }
+    catch(e){
+        console.log(e);
```

Comment:
Nice job Annette, could you please remove all  console.logs, and return the proper response with the appropriate status code